docs/userGuide/syntax/links.mbdf

```diff
@@ -109,11 +109,27 @@ Within `index.md`, we can also display the image using
 
 <div class="indented">
 
-Links will be validated when generating a site and a warning will be displayed in the console for every link that is invalid.
+Links will be validated when generating a site and a warning will be displayed in the console for every link that is invalid. 
+
 
 <box type="warning">
 
-If you wish to disable validation for a specific link, you may use the `no-validation` attribute.
```

Comment:
Let's put the most common feature upfront, then list the more drastic site configuration option second

Reply:
Agreed!

Reply:
Hmm can I check if it is possible to put code blocks into popover / modals? 

![Screen Shot 2021-01-06 at 2 12 33 PM](https://user-images.githubusercontent.com/37920115/103735526-39e32200-5029-11eb-993d-63f4879abcec.png)

I've tried to put the code block into the popover but it seems like it always renders as a normal string, as shown in the image above.

![Screen Shot 2021-01-06 at 2 13 47 PM](https://user-images.githubusercontent.com/37920115/103735625-67c86680-5029-11eb-9350-9a61c77d53ef.png)

Reply:
Try adding a blank line between the div tag and the code block.

Reply:
Thanks Prof! It works now :-)

![Screen Shot 2021-01-06 at 2 32 54 PM](https://user-images.githubusercontent.com/37920115/103736888-140b4c80-502c-11eb-93a1-74fac9bdb319.png)

Does this look better? @ang-zeyu @damithc